IrisBuilders.mjs

```diff
@@ -543,10 +543,6 @@ export function DriverUpdateBuilder(id) {
 }
 
 function getDisplayName(name) {
-    return getDisplayName(name, false);
-}
-
-function getDisplayName(name, onlyFirstCapitalize) {
```

Comment:
@adrianxuny since the intention is for most uses to invoke `getDisplayName(name)` let's just rename the overloaded one (taking the boolean parameter):
```
function getDisplayName(name) {
    return getDisplayNameWithCapitalize(name, false);
}
function getDisplayNameWithCapitalize(name, onlyFirstCapitalize) {
...
```